src/main/java/com/odysseusinc/arachne/portal/repository/BaseStudyRepository.java

```diff
@@ -48,6 +47,7 @@
                     + " studies.id NOT IN (SELECT study_id FROM users_studies_extended "
                     + "                    WHERE user_id=:participantId "
                     + "                    AND lower(status) IN ('pending', 'approved')) "
+                    + " AND studies.kind  <> 'WORKSPACE' "
```

Comment:
Let's use enum constant there to relay on the single point of truth

Reply:
@wivern - I can only add type-safety by adding one more parameter... Please have a look. Is it good enough now?
Btw, I see many other places where strings are used like in the first version